### PR TITLE
feat: add additional metrics to the dashboard

### PR DESCRIPTION
**Pull request by yolocs**
- Added CPU/Mem usage and container instance count metrics
- Uniformly used LINE plotting and fixed some align types

image

### CODE REVIEW

File: terraform/modules/monitoring/dashboard.json [hunk 1]
```diff
@@ -173,8 +277,122 @@
...
+                "minAlignmentPeriod": "60s",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "apiSource": "DEFAULT_CLOUD",
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_PERCENTILE_99",
+                      "perSeriesAligner": "ALIGN_SUM"
```
Comment (raserva) [inline terraform/modules/monitoring/dashboard.json:302]: do we want to sum up the memory usages? This should probably be 99th percentage or avg

**Review by raserva — COMMENTED** [review]
one thing that i notice is that it is a bit hard to differentiate between JVS and certifcate metrics. Is is possible to break them up into separate sections? Or have some other mechanism for logically separating them

File: terraform/modules/monitoring/dashboard.json (hunk 1 above)
Comment (raserva) [reply terraform/modules/monitoring/dashboard.json:302]: I do see the reducer is set to 99th. its not clear to me the difference between reducer and aligner

Comment (yolocs) [reply terraform/modules/monitoring/dashboard.json:302]: Me neither. I simply used the same parameters that cloud run metrics tab has. I suppose it's the good version.
I also didn't find a way to logically separate metrics. Need more learning. Should we get this in first?

Comment (raserva) [reply terraform/modules/monitoring/dashboard.json:302]: Sure, for now we can merge as-is.

**Review by raserva — APPROVED** [review]

**Pull request merged**